frontend/src/components/metrics/ConfigStep/Board/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { InputLabel, ListItemText, MenuItem, Select } from '@mui/material'
+import { BOARD_FIELDS, BOARD_TYPES, emailRegExp, ZERO } from '@src/constants'
+import React, { useState } from 'react'
+import {
+  BoardButtonGroup,
+  BoardForm,
+  BoardSection,
+  BoardTextField,
+  BoardTitle,
+  BoardTypeSelections,
+  VerifyButton,
+} from '@src/components/metrics/ConfigStep/Board/style'
+
+export const Board = () => {
+  const [boardField, seBoardField] = useState({
```

Comment:
Should store those fields into Redux, coc we will use those filed later, like export data